frontend/src/app/components/marker/marker.component.scss

```diff
@@ -1,6 +1,7 @@
 @import '../../../app-theme.scss';
 
 $canvas-size: 600px;
+$spinner-radius: 50px;
```

Comment:
As for why this is 50px, the spinner size is by default 100px as explained here: https://github.com/angular/material2/issues/8835